src/semver.ts

```diff
@@ -1,11 +1,25 @@
-import { promises as fsp } from "node:fs";
-import { resolve } from "node:path";
 import semver from "semver";
 import consola from "consola";
+import {
+  readPackageJSON,
+  resolvePackageJSON,
+  writePackageJSON,
+} from "pkg-types";
 import type { ChangelogConfig } from "./config";
 import type { GitCommit } from "./git";
 
-export type SemverBumpType = "major" | "minor" | "patch";
+export const SemverBumpTypes = {
+  Major: "major",
+  Minor: "minor",
+  Patch: "patch",
+  PreMajor: "premajor",
+  PreMinor: "preminor",
+  PrePatch: "prepatch",
+  PreRelease: "prerelease",
+} as const;
+
+export type SemverBumpType =
+  (typeof SemverBumpTypes)[keyof typeof SemverBumpTypes];
```

Comment:
Keys were starting with lowercase before. Any reason to make them upper case?

Reply:
The `SemverBumpType` is same before
![CleanShot 2023-03-07 at 10 02 59](https://user-images.githubusercontent.com/1111538/223300332-08f71c2d-d3cb-488f-a45b-cf95ad5d88ed.png)

Create `SemverBumpTypes` because use in foreach to check `args`, please check `src/commands/default.ts:119`
![CleanShot 2023-03-07 at 10 04 35](https://user-images.githubusercontent.com/1111538/223300547-2e3cb94c-a98b-421e-8788-7d7783a546a2.png)

Reply:
What is usecase of the object keys?

Reply:
I wish use like enum, so can do:

```
await bumpVersion(commits, config, { type: SemverBumpType.Major })
```